src/GeositeFramework/js/SidebarToggle.js

```diff
@@ -47,20 +47,84 @@
+            this.listenTo(this.viewModel, 'change', this.adjustToggleIcon);
         },
 
         events: {
             'click': 'togglePluginVisibility'
         },
 
+        adjustToggleIcon() {
+            const newIconClass =
+                this.viewModel.get('pluginContentVisible') ?
+                'fa fa-chevron-right' : 'fa fa-chevron-left';
+
+            this.$el.find('i').attr('class', newIconClass);
+        },
+
         togglePluginVisibility: function() {
-            if (this.singlePlugin.selected) {
+            if (this.viewModel.get('pluginContentVisible')) {
+                this.viewModel.set('pluginContentVisible', false);
                 this.singlePlugin.deselect();
-                this.$el.find('i').attr('class', 'fa fa-chevron-right');
             } else {
+                this.viewModel.set('pluginContentVisible', true);
                 this.singlePlugin.select();
-                this.$el.find('i').attr('class', 'fa fa-chevron-left');
             }
         }
     });
+
+    // Mobile toggle view for single plugin mode
+    N.views.MobileTogglePlugin = Backbone.View.extend({
+        initialize: function(options) {
+            this.viewModel = options.viewModel;
+            this.mobileContentButton = $('#single-plugin-toggle-content-button');
+            this.mobileMapButton = $('#single-plugin-toggle-map-button');
+
+            this.listenTo(this.viewModel, 'change', this.adjustButtonCSS);
+            this.showMobileMap = this.showMobileMap.bind(this);
+            this.showMobileContent = this.showMobileContent.bind(this);
+
+            this.mobileMapButton.on('click', this.showMobileMap);
+            this.mobileContentButton.on('click', this.showMobileContent);
+
+            N.app.toggleMobileMap = this.showMobileMap;
+            N.app.toggleMobileContent = this.showMobileContent;
+        },
+
+        adjustButtonCSS: function() {
+            if (this.viewModel.get('pluginContentVisible')) {
+                // Unhighlight mobile map button
+                this.mobileMapButton.addClass('button-secondary');
+                this.mobileMapButton.removeClass('button-primary');
+
+                // Highlight mobile content button
+                this.mobileContentButton.addClass('button-primary');
+                this.mobileContentButton.removeClass('button-secondary');
+            } else {
+                // Highlight mobile map button
+                this.mobileMapButton.removeClass('button-secondary');
+                this.mobileMapButton.addClass('button-primary');
+
+                // Unhighlight mobile content button
+                this.mobileContentButton.removeClass('button-primary');
+                this.mobileContentButton.addClass('button-secondary');
+            }
+        },
+
+        showMobileContent: function() {
+            var singlePlugin = this.viewModel.get('plugins').at(0);
+            singlePlugin.select();
```

Comment:
Can this be omitted? `select` may have side-effects, like reseting controls and options made by the user to the plugin.

Reply:
Let me check on this. My initial thought was that we'd have the buttons behave the same way as the sidebar collapse tab chevron button works in the desktop single web app: it calls `select` and `deselect` when the plugin's expanded and collapsed, respectively.

The idea here was that I wanted to keep them in harmony so now both the `TogglePlugin` view and the `MobileTogglePlugin` view listen for & change on the same model events. 

However, I think you're suggesting that we shouldn't ever deselect the plugin in mobile view -- and instead just show and hide it?

Reply:
That makes sense, but I think we don't actually want it to be in harmony with that button because we want to the plugin to maintain all of it's state in this case. Changes in `select/deselect` don't necessarily reset or lose the plugin state, but I think it can if a plugin chooses to do so. 

Does that sound right @mmcfarland?

Reply:
I updated these to use `show`/`hide` instead of select & deselect.

Reply:
Oof, just realized I updated the wrong set. Pushing another fixup.